checks/raw/ci_tests.go

```diff
@@ -45,19 +44,21 @@ func CITests(c clients.RepoClient) (checker.CITestData, error) {
 
 		prNos[pr.HeadSHA] = pr.Number
 
-		crs, err := c.ListCheckRunsForRef(commit.SHA)
-		if err != nil {
-			return checker.CITestData{}, sce.WithMessage(
-				sce.ErrScorecardInternal,
-				fmt.Sprintf("Client.Repositories.ListCheckRunsForRef: %v", err),
-			)
-		}
+		// HeadSHA is the last commit before the merge. if squashing enabled,
+		// multiple commit SHAs will map to a single HeadSHA
```

Comment:
So in case squashing is not enabled, HeadSHA is the final commit the PR author made. Correct?